Application/targets/STM32/Core/Src/main.c

```diff
@@ -24,6 +24,7 @@
 /* USER CODE BEGIN Includes */
 
 #include "../../Drivers/delay_us.h"
```

Comment:
Should be able to modify the include path settings in the project and then replace this with `#include "delay_us.h`